etc/config/c++.defaults.properties

```diff
@@ -25,6 +25,7 @@ compiler.gdefault.name=g++ default
 
 group.clang.compilers=clang7:clangdefault
 group.clang.intelAsm=-mllvm --x86-asm-syntax=intel
+group.clang.compilerType=clang
```

Comment:
Whoops: thanks for fixing this. I run with a local properties file that overrides this so hadn't spotted it!